R/zzz_distros.R

```diff
@@ -0,0 +1,75 @@
+#' @export
+r2dtable <- rrejig(stats::r2dtable)
+
+#' @export
+rbeta <- rrejig(stats::rbeta)
+
+#' @export
+rbinom <- rrejig(stats::rbinom)
+
+#' @export
+rcauchy <- rrejig(stats::rcauchy)
+
+#' @export
+rchisq <- rrejig(stats::rchisq)
+
+#' @export
+rexp <- rrejig(stats::rexp)
+
+#' @export
+rf <- rrejig(stats::rf)
+
+#' @export
+rgamma <- rrejig(stats::rgamma)
+
+#' @export
+rgeom <- rrejig(stats::rgeom)
+
+#' @export
+rhyper <- rrejig(stats::rhyper)
+
+#' @export
+rlnorm <- rrejig(stats::rlnorm)
+
+#' @export
+rlogis <- rrejig(stats::rlogis)
+
+#' @export
+rmultinom <- rrejig(stats::rmultinom)
+
+#' @export
+rnbinom <- rrejig(stats::rnbinom)
+
+#' @export
+rnorm <- rrejig(stats::rnorm)
+
+#' @export
+rpois <- rrejig(stats::rpois)
+
+#' @export
+rsignrank <- rrejig(stats::rsignrank)
+
+#' @export
+rsmirnov <- rrejig(stats::rsmirnov)
+
+#' @export
+rstandard <- rrejig(stats::rstandard)
```

Comment:
Do you think it makes it much easier to use if you force overwrite like this? Can you instead make it an option that a user applies or similar?

Reply:
In particular how are CRAN going to feel about this

Reply:
Oh, check hates this for sure.

So I think if I 1) tweak them to be in the hashprng package environment, 2) document them all this should be fine. Might need to switch to dispatch to the functions rather than simply duplicating their internals.

Reply:
And yes, definitely easier to use in terms of updating code.

Reply:
> Might need to switch to dispatch to the functions rather than simply duplicating their internals.

That also just seems like a good idea?

Reply:
yeah, different flavor of NSE pita; having a go at it now. longer term, this part would likely pivot to being on the C side anyway for performance, so kind of irrevelant at this stage.